src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleAuthorityRecordCollection.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.inventory.storage.external;
+
+import java.io.IOException;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.folio.Authority;
+import org.folio.dbschema.ObjectMapperTool;
+import org.folio.inventory.domain.AuthorityRecordCollection;
+import org.folio.inventory.validation.exceptions.JsonMappingException;
+
+public class ExternalStorageModuleAuthorityRecordCollection
+  extends ExternalStorageModuleCollection<Authority>
+  implements AuthorityRecordCollection {
+
+  private static final Logger LOGGER = LogManager.getLogger(ExternalStorageModuleAuthorityRecordCollection.class);
+
+  ExternalStorageModuleAuthorityRecordCollection(
+    String baseAddress,
+    String tenant,
+    String token,
+    HttpClient client) {
+
+    super(String.format("%s/%s", baseAddress, "holdings-storage/holdings"),
```

Comment:
Do we need to create endpoints for authority-storage?
Like this [holdings-storage API](https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/holdings-storage.html)

Reply:
Not now.

Reply:
https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/authorities.html
We have them already.

Reply:
We definitely need to rename it to authority-storage like others 😅